snark-verifier/src/pcs/ipa/multiopen/bgh19.rs

```diff
@@ -18,6 +18,8 @@ use std::{
     marker::PhantomData,
 };
 
+/// Verifier of multi-open inner product argument. It is for the implementation
+/// in [`halo2_proofs`].
```

Comment:
Expand Bgh19 for clarity.
```suggestion
/// in [`halo2_proofs`]. <https://eprint.iacr.org/2019/1021>
```